### PR TITLE
Clarify the main/static schema separation, fix some typo

### PR DESCRIPTION
**Pull request by sbrunner**
Fix  #10849

### CODE REVIEW

File: doc/integrator/database.rst [hunk 1]
```diff
@@ -3,6 +3,16 @@
 Database
 ========
 
+Schema
+------
+
+The database table are separate in different schemas:
```
Comment (jwkaltz) [inline doc/integrator/database.rst:9]: Proposal:

The database tables are separated in two schemas:

File: doc/integrator/database.rst [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+- The 'main' schema, can be configured with the ``PGSCHEMA`` environment variable, contains the data that can be modified only by the administrator interface.
```
Comment (jwkaltz) [inline doc/integrator/database.rst:10]: The 'main' schema, which can be configured...

File: doc/integrator/database.rst [hunk 3]
```diff
@@ -3,6 +3,16 @@
...
+- The 'static' schema, can be configured with the ``PGSCHEMA_STATIC`` environment variable, contains the data that can be modified by the end used through the web application.
```
Comment (jwkaltz) [inline doc/integrator/database.rst:11]: The 'static' schema, which can be configured...

Comment (jwkaltz) [inline doc/integrator/database.rst:11]: I don't understand what is meant, which data? What data can be modified through the web application and not via administrator interface? Do you mean the editing functionality?

File: doc/integrator/docker.rst [hunk 4]
```diff
@@ -166,7 +166,7 @@ Config:
 
  * ``VISIBLE_WEB_HOST``: The web host visible by the browser e.g.: 'example.com'.
  * ``VISIBLE_ENTRY_POINT``: The web path visible by the browser e.g.: '/main/', default to '/'.
- * ``PGSCHEMA``: The postgres schema, used by MapServer.
+ * ``PGSCHEMA``: The PostgreSQL schema, used by MapServer.
```
Comment (jwkaltz) [inline doc/integrator/docker.rst:169]: Why do we say "used by MapServer" here? It's hard to understand how it relates to what we say above in the database.rst

File: doc/integrator/database.rst [hunk 5]
```diff
@@ -3,6 +3,17 @@
 Database
 ========
 
+Schema
+------
+
+The database tables are separated in two schemas:
+
+* The 'main' schema, which can be configured with the ``PGSCHEMA`` environment variable, contains the data that can be modified only by the administrator interface.
+* The 'static' schema, can be configured with the ``PGSCHEMA_STATIC`` environment variable, contains the data that can be modified by the end used through the web application (user password, short links, OAuth token and audit logs, not the editing data).
```
Comment (jwkaltz) [inline doc/integrator/database.rst:12]: The static schema, which ...

the end used -> the end user

File: doc/integrator/database.rst [hunk 6]
```diff
@@ -39,12 +50,12 @@ Starting point is that the current version is the same on integration and produc
 Prepare the project
 ~~~~~~~~~~~~~~~~~~~
 
-To be able to proceed like this, the variables ``PGSCHEMA`` and the ``DOCKER_PGSCHEMA_STATIC``
-should be managed in your makefiles:
+To be able to proceed like this, the variables ``PGSCHEMA`` and the ``PGSCHEMA_STATIC``
```
Comment (jwkaltz) [inline doc/integrator/database.rst:53]: and the -> the

File: doc/integrator/database.rst [hunk 7]
```diff
@@ -39,12 +50,12 @@ Starting point is that the current version is the same on integration and produc
...
+should be managed in your make files:
```
Comment (jwkaltz) [inline doc/integrator/database.rst:54]: Are we still using makefiles, I thought we now use env files for this?

**Review by jwkaltz — APPROVED** [review]

File: doc/integrator/database.rst (hunk 7 above)
Comment (sbrunner) [reply doc/integrator/database.rst:54]: Oups, ..., thanks :-)

**Pull request merged**